.travis.yml

```diff
@@ -8,32 +8,29 @@ cache: ccache
 compiler: gcc
 
 notifications:
-  email:
-    recipients:
+  email: true
 
 env:
   global:
     - MOVEIT_CI_TRAVIS_TIMEOUT=85  # Travis grants us 90 min, but we add a safety margin of 5 min
     - ROS_DISTRO=eloquent
     - ROS_REPO=ros
     - UPSTREAM_WORKSPACE=moveit2.repos
-    # moveit_ros_perception: mesh_filter_test fails due to broken Mesa OpenGL
-    - TEST_BLACKLIST="moveit_ros_perception"
     - CXXFLAGS="-Wall -Wextra -Wwrite-strings -Wunreachable-code -Wpointer-arith -Wredundant-decls"
     - WARNINGS_OK=false
-  matrix:
-    # TODO(henningkayser): Enable ament_lint once all packages are ported
-    - TEST="clang-format"
-    - ROS_DISTRO=eloquent
 
-# TODO(mlautman): Enable once franka_ikfast_plugin and panda_ikfast_plugin are available
-# matrix: # Add a separate config to the matrix, using clang as compiler
-  # include:
-    # # add a config to the matrix using clang as compiler and also test ikfast plugin creation
-    # - compiler: clang
-      # env: ROS_REPO=ros
+jobs:
+  fast_finish: true
+  include:
+    - env: TEST="clang-format"  # TODO(henningkayser): Enable ament_lint, clang-tidy
+    - env: TEST=code-coverage
+    - env: ROS_DISTRO=eloquent
+    - compiler: clang  # test_ikfast_plugins takes ~10 minutes: include here to keep the main jobs shorter
+      env: CXXFLAGS="-Wall -Wextra -Wwrite-strings -Wunreachable-code -Wpointer-arith -Wredundant-decls -Wno-overloaded-virtual"
+           # TODO(mlautman): Enable once franka_ikfast_plugin and panda_ikfast_plugin are available
```

Comment:
Unrelated to this PR, but @mlautman can this test be re-enabled now?

Reply:
They should be enabled with this https://github.com/ros-planning/moveit2/pull/205